src/tools/wasm2wast.cc

```diff
@@ -72,8 +72,15 @@ static void parse_options(int argc, char** argv) {
         s_outfile = argument;
         ConvertBackslashToSlash(&s_outfile);
       });
+  // TODO(karlschimpf) Figure out why fold-exprs is on by default.
   parser.AddOption('f', "fold-exprs", "Write folded expressions where possible",
                    []() { s_write_wat_options.fold_exprs = true; });
+  parser.AddOption('n', "no-fold-exprs", "Don't fold expressions",
```

Comment:
I'd rather not add this for now, since it's the default. At least not with a short letter.